common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -827,9 +824,7 @@ public MergeStatuses finalizeShuffleMerge(FinalizeShuffleMerge msg) {
                 msg.appAttemptId, msg.shuffleId, msg.shuffleMergeId, partition.reduceId,
                 ioe.getMessage());
           } finally {
-            AppShufflePartitionInfo.closeAllFilesAndDeleteIfNeeded(false, partition.dataFile,
-                partition.getDataChannel(), partition.getIndexFile(), partition.getMetaFile(),
-                partition.getAppAttemptShuffleMergeId(), partition.getReduceId());
+            partition.cleanable.clean();
```

Comment:
`partitionInfo.deleteAllFiles();` ?

Reply:
The original place where `closeAllFilesAndDeleteIfNeeded(false)` was, doesn't need to call `deleteAllFiles`.

Reply:
I see. Thank you!